boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853_defconfig

```diff
@@ -8,13 +8,8 @@ CONFIG_ARM=y
 CONFIG_SOC_MEC1501_HSZ=y
 CONFIG_SOC_SERIES_MEC1501X=y
 CONFIG_BOARD_MEC15XXEVB_ASSY6853=y
-CONFIG_CORTEX_M_SYSTICK=n
-
-# XEC RTOS timer HW frequency is fixed at 32768
-# The driver requires tickless mode and ticks per
-# second to be 32768 for accurate operation.
-CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=32768
-CONFIG_SYS_CLOCK_TICKS_PER_SEC=32768
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_MCHP_XEC_RTOS_TIMER=n
```

Comment:
In this file use the variable name CONFIG_RTOS_TIMER=y, then turn on CONFIG_MCHP_XEC_RTOS_TIMER in Kconfig.defconfig depenindg on the value of CONFIG_RTOS_TIMER.

Reply:
Where do you suggest to define RTOS_TIMER, at difference than other drivers for which CONFIG_GPIO, etc does exist. Timer drivers are fully identified in drivers/timers/KConfig
And I don't think this is a good fit to define in mec soc kconfig.

Reply:
Cleaned up to have single switch.

Reply:
I don't get what's the difference with the other drivers.  However, I agree to use CONFIG_MCHP_XEC_RTOS_TIMER since no one is using a generic flag in their boards for their non-systick timers.

Reply:
Revisit your -1, since we now have single switch RTOS_TIMER option as your original idea, but not exactly the same way.